web/src/games/cardtable/deals.ts

```diff
@@ -1,56 +1,166 @@
+import imgAC from './media/png/AC.png';
+import imgKC from './media/png/KC.png';
+import imgQC from './media/png/QC.png';
+import imgJC from './media/png/JC.png';
+import img10C from './media/png/10C.png';
+import img9C from './media/png/9C.png';
+import img8C from './media/png/8C.png';
+import img7C from './media/png/7C.png';
+import img6C from './media/png/6C.png';
+import img5C from './media/png/5C.png';
+import img4C from './media/png/4C.png';
+import img3C from './media/png/3C.png';
+import img2C from './media/png/2C.png';
+import imgAS from './media/png/AS.png';
+import imgKS from './media/png/KS.png';
+import imgQS from './media/png/QS.png';
+import imgJS from './media/png/JS.png';
+import img10S from './media/png/10S.png';
+import img9S from './media/png/9S.png';
+import img8S from './media/png/8S.png';
+import img7S from './media/png/7S.png';
+import img6S from './media/png/6S.png';
+import img5S from './media/png/5S.png';
+import img4S from './media/png/4S.png';
+import img3S from './media/png/3S.png';
+import img2S from './media/png/2S.png';
+import imgAD from './media/png/AD.png';
+import imgKD from './media/png/KD.png';
+import imgQD from './media/png/QD.png';
+import imgJD from './media/png/JD.png';
+import img10D from './media/png/10D.png';
+import img9D from './media/png/9D.png';
+import img8D from './media/png/8D.png';
+import img7D from './media/png/7D.png';
+import img6D from './media/png/6D.png';
+import img5D from './media/png/5D.png';
+import img4D from './media/png/4D.png';
+import img3D from './media/png/3D.png';
+import img2D from './media/png/2D.png';
+import imgAH from './media/png/AH.png';
+import imgKH from './media/png/KH.png';
+import imgQH from './media/png/QH.png';
+import imgJH from './media/png/JH.png';
+import img10H from './media/png/10H.png';
+import img9H from './media/png/9H.png';
+import img8H from './media/png/8H.png';
+import img7H from './media/png/7H.png';
+import img6H from './media/png/6H.png';
+import img5H from './media/png/5H.png';
+import img4H from './media/png/4H.png';
+import img3H from './media/png/3H.png';
+import img2H from './media/png/2H.png';
+import imgGB from './media/png/gray_back.png';
+
+export const deckAssets = {
+  AS: imgAS,
+  '2S': img2S,
```

Comment:
You can use an enum for the keys too, so you get type safety:

interface Foo {
   [key: NewEnum]: Bar;
} 
let foo:Foo = {};

Reply:
so I'm a little green in TS I'll have to look at your example.. I've worked with enums in other languages, just not JS/TS...

Reply:
not sure if I got this completely right, it is addressed in forthcoming commit.